app/javascript/components/collections/CollectionsDetail.vue

```diff
@@ -78,6 +78,8 @@
         <CollectionsRepoList
           v-if="showRepoList"
           :repositories="collectionData.repositories"
+          :collectionsId="collectionsId"
+          :settingsVisibility="settingsVisibility"
```

Comment:
Please update the name 'settingsVisibility' to  'canManage'

Reply:
done